src/Controller/DefaultController.php

```diff
@@ -15,5 +16,10 @@ class DefaultController extends FrontendController
     public function defaultAction(Request $request): Response
     {
         return $this->render('default/default.html.twig');
+    }    
+    
+    public function loginAction(): Response
+    {
+        return $this->forward(LoginController::class.'::loginAction');
```

Comment:
```suggestion
        return $this->forward(LoginController::class.'::loginCheckAction');
```
Even better as this authenticates active session